ascend-sdk/src/main/java/ai/evolv/AscendClientFactory.java

```diff
@@ -55,6 +51,7 @@ private static AscendClient createClient(AscendConfig config, AscendParticipant
                 new EventEmitter(config, participant),
                 futureAllocations,
                 allocator,
-                reconciliationNeeded);
+                reconciliationNeeded,
```

Comment:
not sure if reconilationNeeded is used elsewhere but it seems like this is really just Allocator.allocationsNotEmpty(previousAllocations)

Reply:
Good point!